packages/core/admin/admin/src/content-manager/history/components/VersionContent.tsx

```diff
@@ -181,7 +181,7 @@ const VersionContent = () => {
                             return JSON.stringify(value);
                           case 'date':
                           case 'datetime':
-                            return new Date(value as string);
+                            return value;
```

Comment:
ℹ️ Without this change everything is broken but not sure it's the actual solution. I'll double check tomorrow.